lib/p2p/Pool.ts

```diff
@@ -928,11 +928,16 @@ class Pool extends EventEmitter {
     peer.on('reputation', async (event) => {
       this.logger.debug(`Peer (${peer.label}): reputation event: ${ReputationEvent[event]}`);
       if (peer.nodePubKey) {
-        if (this.testing && event !== ReputationEvent.ManualBan && event !== ReputationEvent.ManualUnban) {
-          // we don't add non-manual reputation events when in debug/testing mode to prevent unintentional bans
-          return;
+        // we only add manual reputation events when not in strict mode to prevent unintentional bans
+        if (this.strict
```

Comment:
Should this be `!this.strict` so it's in line with the comment above?

Reply:
The `if` statement is correct, but the comment isn't very clear, so I'll reword the comment. We want to add reputation events when in strict mode OR if the reputation event is manual or severe (which we add in both strict and non strict modes).